docs/dashboard-operator.md

```
@@ -46,26 +45,22 @@ The CRD embeds types from `odh-platform-utilities/api/common`:
 
 ## Reconciliation Pipeline
 
-The controller follows a sequential pipeline on each reconcile, branching based on the `deploymentMode` spec field:
+The controller follows a sequential pipeline on each reconcile:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)